src/codegen/llvm/codegen_llvm.cc

```diff
@@ -567,6 +567,12 @@ llvm::Value* CodeGenLLVM::CreateIntrinsic(const Call* op) {
     } else {
       return builder_->CreateLShr(MakeValue(op->args[0]), MakeValue(op->args[1]));
     }
+  } else if (op->is_intrinsic(Call::popcount)) {
```

Comment:
instead of doing this,  use https://github.com/dmlc/tvm/blob/master/src/codegen/llvm/intrin_rule_llvm.cc